src/hooks/use-sync-states-progress-info.ts

```diff
@@ -30,12 +35,42 @@ export function useSyncStatesProgressInfo() {
 			finished: {
 				key: 'finished',
 				progress: 100,
-				message: __( 'Backup imported' ),
+				message: __( 'Pull complete' ),
+			},
+			failed: {
+				key: 'failed',
+				progress: 100,
+				message: __( 'Error pulling changes' ),
+			},
+		} as const;
+	}, [ __ ] );
+
+	const pushStatesProgressInfo = useMemo( () => {
+		return {
+			creatingBackup: {
+				key: 'creatingBackup',
+				progress: 30,
+				message: __( 'Creating backup…' ),
+			},
+			uploading: {
+				key: 'uploading',
+				progress: 50,
+				message: __( 'Pushing changes…' ),
+			},
+			importing: {
+				key: 'importing',
+				progress: 80,
+				message: __( 'Pushing changes…' ),
```

Comment:
Not sure what message we could use here. Any thoughts?

Reply:
This is the only message that appears in the design. I think it's good to leave it.
Other options:
- Applying changes
- Unpacking changes

Reply:
We could also split this state using the remote states, so the user will get more information `initial_backup_started`, `archive_import_started`, etc.

Reply:
'Applying changes' sounds good for now.

Reply:
> there is not system notification after the finished Push, but we have one for Pull. We can add a separate issue for that, though.
I implemented it in this PR 🥳: https://github.com/Automattic/studio/pull/662/commits/8e22d1b2acd3202395b2e502863d4edfa641e7c7

> 'Applying changes' sounds good for now.
- I've created the issue: https://github.com/Automattic/dotcom-forge/issues/9847